style.css

```diff
@@ -24,7 +24,62 @@ body {
 #tasksNav {
   background-color: var(--blue-color);
   padding: 10px;
-  height: 2.5rem;
+  display: flex;
+}
+
+img {
+  height: 100%;
+  width: 100%;
+}
```

Comment:
Modifying global css doesn't seem right. What do you think?

Reply:
I think it will protect every images from overflow and i couldn't think of any problems.